packages/reports/addon/templates/components/power-select-bulk-import-trigger.hbs

```diff
@@ -1,46 +1,59 @@
-  {{#each select.selected as |opt idx|}}
-    <li class="ember-power-select-multiple-option {{if opt.disabled "ember-power-select-multiple-option--disabled" }}">
-      {{#unless select.disabled }}
+
+<ul
+  id="ember-power-select-multiple-options-{{@select.uniqueId}}"
+  class="ember-power-select-multiple-options"
+  {{did-update this.openChanged @select.isOpen}}
+  {{on "touchstart" this.chooseOption}}
+  {{!-- template-lint-disable no-invalid-interactive no-positive-tabindex --}}
+  {{on "mousedown" this.chooseOption}}
+  ...attributes>
+  {{#each @select.selected as |opt idx|}}
+    <li class="ember-power-select-multiple-option {{if opt.disabled "ember-power-select-multiple-option--disabled"}}">
+      {{#unless @select.disabled}}
         <span role="button"
           aria-label="remove element"
           class="ember-power-select-multiple-remove-btn"
           data-selected-index={{idx}}>
           &times;
         </span>
       {{/unless}}
-      {{#if selectedItemComponent}}
-        {{component selectedItemComponent option=(readonly opt) select=(readonly select)}}
+      {{#if @selectedItemComponent}}
+        {{component @selectedItemComponent extra=@extra option=opt select=@select}}
       {{else}}
-        {{yield opt select}}
+        {{yield opt @select}}
       {{/if}}
     </li>
   {{else}}
-    {{#if (and placeholder (not searchEnabled))}}
-      <span class="ember-power-select-placeholder">{{placeholder}}</span>
+    {{#if (and @placeholder (not @searchEnabled))}}
+      <span class="ember-power-select-placeholder">{{@placeholder}}</span>
     {{/if}}
   {{/each}}
-  {{#if searchEnabled}}
+  {{#if @searchEnabled}}
     {{!-- template-lint-disable no-positive-tabindex --}}
-    <input type="search" class="ember-power-select-trigger-multiple-input"
-      autocomplete="new-password" autocorrect="off" autocapitalize="off" spellcheck="false"
-      id="ember-power-select-trigger-multiple-input-{{select.uniqueId}}"
-      value={{select.searchText}}
-      aria-controls={{listboxId}}
-      style={{triggerMultipleInputStyle}}
-      placeholder={{maybePlaceholder}}
-      disabled={{select.disabled}}
-      oninput={{action "onInput"}}
-      onPaste={{action "onPaste"}}
-      onFocus={{onFocus}}
-      onBlur={{onBlur}}
-      tabindex={{tabindex}}
-      onkeydown={{action "onKeydown"}}>
+    <input
+      type="search"
+      class="ember-power-select-trigger-multiple-input"
+      autocomplete="new-password"
+      autocorrect="off"
+      autocapitalize="off"
+      spellcheck={{false}}
+      id="ember-power-select-trigger-multiple-input-{{@select.uniqueId}}"
+      value={{@select.searchText}}
+      aria-controls={{@listboxId}}
+      style={{this.triggerMultipleInputStyle}}
+      placeholder={{this.maybePlaceholder}}
+      disabled={{@select.disabled}}
+      tabindex={{@tabindex}}
+      {{!-- TODO Paste has been disabled since bulk import is not working --}}
+      {{!-- onPaste={{action "onPaste"}} --}}
```

Comment:
reenable?

Reply:
Ok mis understood why master branch isn't working. This PR wasn't working because template location issue. Fixed it.